engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailPendoResolver.java

```diff
@@ -12,14 +12,21 @@ public class EmailPendoResolver {
     public static String GENERAL_PENDO_MESSAGE = "You can also receive notifications on tools like Slack, Microsoft Teams, Google chat, and more! Go to <b><a href=\"%s/settings/integrations\" target=\"_blank\">integrations</a></b> to set those up.";
     public static String GENERAL_PENDO_TITLE = "Did you know?";
 
+    public static final String OUTAGE_PENDO_MESSAGE = "Between Aug 22, 07:21 AM UTC and Aug 26, 12:47 PM UTC, the notifications email service experienced an outage. This email was delayed from that time period.";
```

Comment:
The initial analysis showed the incident started at 07:21 but we have a few emails affected between 07:19 and 07:21. Let's use the exact time in the message.
```suggestion
    public static final String OUTAGE_PENDO_MESSAGE = "Between Aug 22, 07:19 AM UTC and Aug 26, 12:47 PM UTC, the notifications email service experienced an outage. This email was delayed from that time period.";
```

Reply:
Ok sure